src/main/java/io/kestra/plugin/gcp/firestore/Set.java

```diff
@@ -28,19 +28,38 @@
     examples = {
         @Example(
             title = "Set a document from a map.",
-            code = {
-                "collection: \"persons\"",
-                "document:",
-                "  firstname: \"John\"",
-                "  lastname: \"Doe\"",
-            }
+            full = true,
+            code = """
+                id: gcp_firestore_set
+                namespace: company.name
+
+                tasks:
+                  - id: set
+                    type: io.kestra.plugin.gcp.firestore.Set
+                    collection: "persons"
+                    document:
+                      firstname: "John"
+                      lastname: "Doe"
+                """
         ),
         @Example(
             title = "Set a document from a JSON string.",
-            code = {
-                "collection: \"persons\"",
-                "document: \"{{ outputs.task_id.data | json }}\""
-            }
+            full = true,
+            code = """
+                id: gcp_firestore_set
+                namespace: company.name
+
+                inputs:
+                  - id: json_string
+                    type: STRING
+                    default: "{\"firstname\": \"John\", \"lastname\": \"Doe\"}"
```

Comment:
Does this need to be escaped inside `"""`? @shrutimantri

Reply:
I added one more double quotes around it as there are `:` inside.